src/connector.ts

```diff
@@ -70,6 +72,13 @@ export class Connector {
     return data.items;
   }
 
+  public async fetchQuery<T extends FDMQueryResponse>(
+    params: RequestParams
+  ): Promise<FDMQueryResponse> {
+    const { data } = await this.fetchData<Response<T>>(params);
```

Comment:
```suggestion
    const { data } = await this.fetchData<DataResponse<T>>(params);
```
and revert the response type back